src/libs/Dongle.js

```diff
@@ -0,0 +1,95 @@
+import ledger from "ledgerco";
```

Comment:
it's still the previous library?

Reply:
We will eventually switch to ledgerjs when support fro segwit, timestamp and different network is implemented.